app/schema/file.py

```diff
@@ -0,0 +1,56 @@
+import enum
+from pydantic import BaseModel, ConfigDict
+
+
+class FileType(enum.Enum):
+    IMAGE = "image"
+    VIDEO = "video"
+    ATTACHMENT = "attachment"
+    AVATAR = "avatar"
+    UNKNOWN = "unknown"
+
+
+class File(BaseModel):
+    name: str = ""
+    original_name: str = ""
+    type: str = FileType.IMAGE.value
+    url: str
+
+    model_config = ConfigDict(
+        from_attributes=True,
+    )
+
+
+class FileIn(BaseModel):
+    name: str
+    file_base64: str
```

Comment:
why do we need base_64 of file????

Reply:
We used to have base 64 file upload approach since there were problems with file uploads in next js server. I think we should use default file upload approach of FastAPI